rocky/components/modal/script.js

```diff
@@ -10,67 +21,52 @@ export function initDialogs(element) {
 }
 
 export function initDialog(modal) {
-  let input_elements = [];
+  let dialog_element = modal.querySelector("dialog");
+  if (!dialog_element) return;
 
-  modal.querySelector(".modal-trigger").addEventListener("click", (event) => {
-    // Get and clone input elements to be able to "reset" them on "cancel".
-    input_elements = modal.querySelectorAll("input, textarea");
+  let trigger = document.querySelector(
```

Comment:
Im thinking we could probably do a :not(a) here and remove that check on line 33.